src/KeyFrame.cc

```diff
@@ -339,6 +352,25 @@ std::tuple<double,double,double> KeyFrame::MakeKeyFrame_Lite(CVD::Image<CVD::byt
         fast_corner_detect_10(lev.image, lev.vCorners, 10);
         lev.nFastThresh = 10;
       }
+      
+        //do nonmax suppression
+  
+	  std::vector<CVD::ImageRef> vMaxCornersTemp;
+      CVD::fast_nonmax(lev.image, lev.vCorners, lev.nFastThresh, vMaxCornersTemp);
+          
+      std::vector<int> vMaxScoresTemp;
+      CVD::fast_corner_score_10(lev.image, vMaxCornersTemp, lev.nFastThresh, vMaxScoresTemp);  
+      
+      ROS_ASSERT(vMaxScoresTemp.size() == vMaxCornersTemp.size());
```

Comment:
Do we want ASSERTS? I thought we opted to handle errors instead of crashing MCPTAM if something fails.